packages/grafana-e2e/src/plugins/settings.ts

```diff
@@ -1,12 +1,14 @@
-import { PluginMeta } from '@grafana/ui';
+// Not sure yet if e2e should import from g/ui
+// import { PluginMeta } from '@grafana/ui';
```

Comment:
WDYT @ryantxu ?

Reply:
what is the downside?  that it is fixed at a particular version?

I would like the main package to include common tests that should run everywhere -- regardless of version:
* did plugin load
* show the plugin page -- and check for errors